src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function(firstDigit) {
+    return function(secondDigit) {
+      if (counter >= 3 && counter % 2 === 1) {
+        counter++;
+
+        return 'Bzzz... Error!';
+      }
+
+      counter++;
+
+      return firstDigit + secondDigit;
```

Comment:
```suggestion
      counter++;

      return counter > 3 && counter % 2 === 0
        ? 'Bzzz... Error!'
        : firstDigit + secondDigit;
```
you dont need to repeat counter++ 2 times

Reply:
> you dont need to repeat counter++ 2 times

fixed. ty